src/olympia/addons/tests/test_serializers.py

```diff
@@ -1543,7 +1543,7 @@ def test_current_compatible_version(self):
         # Create a new current version, just to prove that
         # current_compatible_version does not use that.
         version_factory(addon=self.addon)
-        self.addon.reload
+        self.addon.reload()
```

Comment:
:see_no_evil:

Reply:
heh, fortunately the only instance I found